assets/js/components/dashboard-sharing/DashboardSharingDialog/Footer/index.js

```diff
@@ -133,7 +133,11 @@ export default function Footer( { closeDialog, openResetDialog } ) {
 					settingsDialogOpen &&
 					! showNotice && (
 						<div className="googlesitekit-dashboard-sharing-settings__footer-actions-left">
-							<Link onClick={ openResetDialog } danger>
+							<Link
+								onClick={ openResetDialog }
+								danger
+								className="googlesitekit-reset-sharing-permissions-button"
```

Comment:
We need to change the order of properties. The first one should be `className`, the second `onClick` and the last one should be `danger` because properties with no values should always go at the end.